src/components/TodoList.jsx

```diff
@@ -0,0 +1,34 @@
+import Todo from "./Todo";
+import { useSelector } from "react-redux";
+import { taskDone, taskDeleted, taskUpdated } from "../features/todos";
+import { useDispatch } from "react-redux";
+
+export default function TodoList() {
+  const todos = useSelector((state) => state.todos);
+  const dispatch = useDispatch();
+
+  const createCompleteHandler = (id) => {
+    return () => dispatch(taskDone(id));
+  };
+  const createDeleteHandler = (id) => {
+    return () => dispatch(taskDeleted(id));
+  };
+  const createUpdateHandler = (id) => {
+    return (text) => dispatch(taskUpdated({ id, text }));
+  };
+
```

Comment:
I believe that including these functions in the todo.jsx file would be a great way to minimize the need for unnecessary functions and props